### ISSUE TITLE
Key position markers sometimes hidden

### ISSUE THREAD

**Issue by donkirkby** [cfe-lab/MiCall#277]
If a region is too wide, the marker for a single key position may be thinner than a single pixel. One example is the HCV1A-H77-NS3 region.

To fix it, just calculate the minimum width for a marker rectangle as a fraction of the region's width.

This issue may not matter after #344 is complete. Check to see if we have a similar problem in the Python coverage maps.

**Comment by donkirkby** [comment]
No longer relevant, since switching to Python coverage maps in issue #344.
